bin/verilator

```diff
@@ -4718,7 +4731,7 @@ correctly.
 
 Warns that due to some construct, optimization of the specified signal or
 block is disabled.  The construct should be cleaned up to improve
-simulation runtime.
+simulation performance.
```

Comment:
Above you used "speed" perhaps we should unify on 'performance'?

Reply:
I think standard terminology is be great, though in some instances (like this) a more specific term often helps people who are not native speakers (i.e.: myself). I normally try to match the more specific adjectives: faster <-> speed, improve <-> performance, though I see I am not doing it consistently. If you want to unify I'm happy with anything but "runtime" due to the ambiguity about it meaning "the length of time it takes to perform the simulation" vs "the point in time when you perform the simulation". I know the context disambiguates but it's easier not to have to think about it.

Reply:
Whatever wording you think reads well.